examples/getting_started/pt/nvflare_pt_getting_started.ipynb

```diff
@@ -0,0 +1,412 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7a5c3d67-a6ea-4f59-84d2-effc3ef016e1",
+   "metadata": {},
+   "source": [
+    "# Getting Started with NVFlare (PyTorch)\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/NVIDIA/NVFlare/blob/job_tutorial/examples/hello-world/job_api/pt/nvflare_job_getting_started.ipynb)\n",
```

Comment:
We should update Colab link for new location and without branch name

Reply:
Good catch! Updated.